scripts/generate-google3-docs/resource-reference/overview.md

```diff
@@ -842,6 +842,10 @@ issues for {{product_name_short}}.
   <td><a href="/vpc/docs/">{{serverless_vpc_access_name}}</a></td>
   <td><a href="/config-connector/docs/reference/resource-docs/vpcaccess/vpcaccessconnector.md">VPCAccessConnector</a></td>
 </tr>
+<tr>
```

Comment:
The section should be moved to the place where the name is ordered.

Reply:
We should write a linter!